vnext/Microsoft.ReactNative/ReactHost/ReactInstanceWin.h

```diff
@@ -163,8 +161,6 @@ class ReactInstanceWin final : public Mso::ActiveObject<IReactInstanceInternal,
   std::shared_ptr<facebook::react::MessageQueueThread> m_batchingUIThread;
 
   std::shared_ptr<react::uwp::IReactInstance> m_legacyReactInstance;
-  std::shared_ptr<react::uwp::DeviceInfo> m_deviceInfo;
```

Comment:
>m_deviceInfo [](start = 42, length = 12)

Removing the explicit dependency on specific module from the ReactInstance looks awesome!